.npmignore

```diff
@@ -0,0 +1,47 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+
+# Runtime data
+pids
+*.pid
+*.seed
+
+# Directory for instrumented libs generated by jscoverage/JSCover
+lib-cov
+
+# Coverage directory used by tools like istanbul
+coverage
+
+# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
+.grunt
+
+# node-waf configuration
+.lock-wscript
+
+# Compiled binary addons (http://nodejs.org/api/addons.html)
+build/Release
+
+# Optional npm cache directory
+.npm
+
+# Optional REPL history
+.node_repl_history
+
+.DS_Store
+
+*.swp
+todo.txt
+test.js
+output.txt
+output/
+
+examples/
+icons/
+dist/
```

Comment:
oh but not dist, right?

Reply:
and test?

Reply:
Oh yeah, I'll add test! 
Do you think we need dist in the npm distribution, I was thinking maybe we can distribute the dist file separately using a cdn, kind of like jquery, what say?